modules/installation-gcp-user-infra-wait-for-bootstrap.adoc

```diff
@@ -44,8 +44,8 @@ has initialized.
 +
 [source,terminal]
 ----
-$ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-instance-group --instance-group-zone=${ZONE_0}
-$ gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
-$ gsutil rb gs://${INFRA_ID}-bootstrap-ignition
+$ gcloud compute backend-services remove-backend ${INFRA_ID}-api-internal-backend-service --region=${REGION} --instance-group=${INFRA_ID}-bootstrap-ig --instance-group-zone=${ZONE_0}
+$ gsutil rm gs://${INFRA_ID}--bootstrap-ignition/bootstrap.ign
```

Comment:
Sorry, please remove the additional hyphen, thanks!

Reply:
Thanks, @jianli-wei . Updated.